product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -4,57 +4,57 @@ title: Maintenance commands through proxies
 
 ## Maintenance and performance
 
-As a general rule, you should never perform maintenance operations on a cluster's write leader. 
+As a general rule, don't perform maintenance operations on a cluster's write leader. 
```

Comment:
No - You Should Never. This is very important emphasis.

Reply:
```suggestion
As a general rule, you should never perform maintenance operations on a cluster's write leader. 
```